lib/index.js

```diff
@@ -51,7 +51,7 @@ function factory(tree, options) {
     if (left && 'data' in left) {
```

Comment:
```suggestion
    if (left && left.data) {
```

How about this instead?

Currently, your proposal will break on the next if statement.